src/main/java/com/minecolonies/api/items/ModItems.java

```diff
@@ -185,6 +183,12 @@ public static Item[] getAllIngredients()
     public static Item[] getAllFoods()
     {
         return new Item[] {
+          // Breads
+          milkyBread,
+          sugaryBread,
+          goldenBread,
+          chorusBread,
+
```

Comment:
I think it's fine if we keep these out as currently of the getAllFoods. They are some "special items" for special uses cases and not regular food.

Reply:
Alright, I wasn't sure if that was intentional or forgotten, I'll remove them